plugin/dap.lua

```diff
@@ -0,0 +1,24 @@
+
+if not vim.api.nvim_create_user_command then
+  return
+end
+
+local cmd = vim.api.nvim_create_user_command
+cmd('DapSetLogLevel',
+  function(opts)
+    require('dap').set_log_level(unpack(opts.fargs))
+  end,
+  {
+    nargs = 1,
+    complete = function()
+      return vim.tbl_keys(require('dap.log').levels)
+    end
+  }
+)
+cmd('DapShowLog', 'split | e ' .. vim.fn.stdpath('cache') .. '/dap.log | normal! G', {})
+cmd('DapContinue', function() require('dap').continue() end, { nargs = 0 })
+cmd('DapToggleBreakpoint', function() require('dap').toggle_breakpoint() end, { nargs = 0 })
+cmd('DapOpenRepl', function() require('dap').toggle_breakpoint() end, { nargs = 0 })
```

Comment:
typo here? action should be
```suggestion
cmd('DapOpenRepl', function() require('dap').repl.open() end, { nargs = 0 })
```

Reply:
Thanks for the pointer.